flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/runtime/functions/ScalarFunctions.scala

```diff
@@ -296,4 +296,10 @@ object ScalarFunctions {
     */
   def repeat(base: String, n: Int): String = EncodingUtils.repeat(base, n)
 
+  /**
+    * Returns the string with the order of characters reversed.
+    */
+  def reverse(base: String): String = {
+    new StringBuffer(base).reverse().toString
+  }
```

Comment:
Good idea, I've changed to `StringBuilder `

Reply:
Using `StringBuilder` may be better.